source/java/src/org/lucee/extension/orm/functions/ORMCloseAllSessions.java

```diff
@@ -4,23 +4,28 @@
  *
  * This library is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either 
+ * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
+ *
  * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
  * Lesser General Public License for more details.
- * 
- * You should have received a copy of the GNU Lesser General Public 
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
not sure what is going on here, but print and aprint are classes present in the lucee core and every extension, they are used to make printout while developing and should NOT be removed. 
- print can be used like the dump function and can better show complex structures than System.out
- you can simply remove the "extends aprint" from the print class to see where the class is used, helpful to clean up

please do not simply remove stuff you don't completely understand the need for